packages/babel-parser/src/plugins/typescript/index.ts

```diff
@@ -3026,7 +3026,10 @@ export default (superClass: ClassWithMixin<typeof Parser, IJSXParserMixin>) =>
           /* isExport */ true,
         );
         if (process.env.BABEL_8_BREAKING) {
+          (node as Undone<N.ExportNamedDeclaration>).attributes = [];
           (node as Undone<N.ExportNamedDeclaration>).declaration = declaration;
+          (node as Undone<N.ExportNamedDeclaration>).exportKind = "value";
+          (node as Undone<N.ExportNamedDeclaration>).source = null;
```

Comment:
Out of curiosity: why is only this node not in `fillOptionalPropertiesForTSESLint`?

Reply:
Good question. This is the situation where we have already decided to completely align AST to ts-eslint: Babel 8 generates the same AST as ts-eslint for import equals.

For other TS nodes, we can decide whether we will align to ts-eslint. If yes, we can definitely remove more cases from `fillOptionalPropertiesForTSESLint`.